pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -1042,35 +1048,41 @@ private QuestionRecord createRecordForComponent(FormResponse formResponse, Strin
      * @param instance The activity instance that has answers
      * @return A question record, or null if it's an unanswered deprecated question that should be skipped for export
      */
-    private QuestionRecord createRecordForQuestion(QuestionDef question, FormResponse instance) {
+    private QuestionRecord createRecordForQuestion(QuestionDef question, FormResponse instance, Handle handle) {
         if (question.isDeprecated() && !instance.hasAnswer(question.getStableId())) {
             return null;
         }
         Answer answer = instance.getAnswer(question.getStableId());
-        if (answer == null) {
+        if (answer == null && question.getQuestionType() != QuestionType.EQUATION) {
             return null;
         }
-        QuestionType type = answer.getQuestionType();
+        QuestionType type = question.getQuestionType();
         if (type == QuestionType.DATE) {
             DateValue value = (DateValue) answer.getValue();
             return new DateQuestionRecord(question.getStableId(), value);
-        } else if (answer.getQuestionType() == QuestionType.FILE) {
+        } else if (type == QuestionType.FILE) {
             List<FileInfo> fileInfos = ((FileAnswer) answer).getValue();
             List<Long> uploadIds = fileInfos == null
                     ? Collections.emptyList() :
                     fileInfos.stream().map(FileInfo::getUploadId).collect(Collectors.toList());
             return new SimpleQuestionRecord(type, question.getStableId(), uploadIds);
-        } else if (answer.getQuestionType() == QuestionType.PICKLIST) {
+        } else if (type == QuestionType.PICKLIST) {
             List<SelectedPicklistOption> selected = ((PicklistAnswer) answer).getValue();
             return new PicklistQuestionRecord(question.getStableId(), selected);
-        } else if (answer.getQuestionType() == QuestionType.MATRIX) {
+        } else if (type == QuestionType.MATRIX) {
             List<SelectedMatrixCell> selected = ((MatrixAnswer) answer).getValue();
             return new MatrixQuestionRecord(question.getStableId(), selected);
-        } else if (answer.getQuestionType() == QuestionType.COMPOSITE) {
+        } else if (type == QuestionType.COMPOSITE) {
             List<AnswerRow> rows = ((CompositeAnswer) answer).getValue();
             return new CompositeQuestionRecord(question.getStableId(), rows);
-        } else if (answer.getQuestionType() == QuestionType.DECIMAL) {
+        } else if (type == QuestionType.DECIMAL) {
             return new SimpleQuestionRecord(type, question.getStableId(), ((DecimalAnswer) answer).getValueAsBigDecimal());
+        } else if (type == QuestionType.EQUATION) {
+            final Optional<QuestionDto> equationDto = handle.attach(QuestionDao.class).getJdbiQuestion()
```

Comment:
May be we can get the answers evaluated as part of ActivityResponse but should be ok for now here.